### DISCUSSION TITLE
Display square cube colors basing on GeoJson layer colors

### DISCUSSION THREAD

**Discussion by n-h-tuan** [CityScope/CS_cityscopeJS#133] [Q&A]
Hi everyone, I am working on this field and having this circumstance. Specifically,  I want to create a GRID with the same color of each cube lying on that geojson layer (see attached image). How can I do that? Thank you.
![1](https://user-images.githubusercontent.com/35844864/117749924-c5212e00-b23c-11eb-86c9-a5be3dc02c00.jpg)

**Comment by RELNO** [comment]
Hi @nguyenledly, thanks for the question. 
In the CityScopeJS Editor (where you create the grid), each row in the table is clickable. If you click it, you can draw the desired color/landuse on your grid. 
![Screen Shot 2021-05-11 at 9 07 59 AM](https://user-images.githubusercontent.com/14296774/117820282-648ff200-b238-11eb-968b-c51342378983.png)

**Reply by n-h-tuan** [reply]
Thanks for reply @RELNO . So there is no way to initialize the grid with the same color of geojson layer? Because I want to make it immediately when clicking button "Create Grid". Thank you.

**Reply by RELNO** [reply]
Sounds like a great feature! We had a few attempts to do that in the past, and the most promising method used sampling of grid centers to understand the geojson. However, it had several issues, which you can read about here:  https://github.com/CityScope/CS_cityscopeJS/issues/110

**Reply by n-h-tuan** [reply]
Thanks for your answer @RELNO I am taking a look on that issue. Hoping in the future we will have this great stuff.
